docs/swagger.yaml

```
@@ -1,6 +1,36 @@
+      name:
+        type: string
+      uuid:
+        type: string
+    type: object
 info:
   contact: {}
 paths:
+  /test:
+    post:
+      consumes:
+      - application/json
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)